data/src/main/kotlin/com/alfresco/content/data/SearchRepository.kt

```diff
@@ -212,15 +216,68 @@ class SearchRepository {
             null
         }
 
-    suspend fun getRecents(skipCount: Int, maxItems: Int) =
-        ResponsePaging.with(
-            searchService.recentFiles(
-                session.account.id,
-                MAX_RECENT_FILES_AGE,
-                skipCount,
-                maxItems,
-            ),
-        )
+    suspend fun getRecents(skipCount: Int, maxItems: Int): ResponsePaging {
+        val version = getServerVersion()
+
+        if (version.toInt() >= 23) {
```

Comment:
Can we store version number 23 into a variable and then use it here?

Reply:
done